src/helpers/pollUpdates.spec.ts

```diff
@@ -15,12 +15,19 @@ describe('poll updates', () => {
   const fetchBackup = fetch;
   let store: Store;
   let dispatch: jest.SpyInstance;
+  const dateBackup = Date;
 
   beforeEach(() => {
     jest.useFakeTimers();
     store = createTestStore();
     dispatch = jest.spyOn(store, 'dispatch');
     (global as any).fetch = mockFetch(200, {release_id: 'releaseid'});
+
+    function MockDate() {
+      return null;
+    }
+    MockDate.prototype.getTime = jest.fn(() => (new dateBackup()).getTime());
+    (global as any).Date = MockDate;
   });
```

Comment:
Tutor's has a handy time mocking helper for jest that you might take a look at: 
https://github.com/openstax/tutor-js/blob/master/tutor/specs/helpers/time-mock.js

It's used like this: https://github.com/openstax/tutor-js/blob/master/tutor/specs/screens/assignment-builder/homework.spec.js#L11

Reply:
i may look into a helper like this if i have to do it again.. but for now i'll leave as is